### ISSUE TITLE
Split Mono.Cairo out

### ISSUE THREAD

**Issue by decriptor** [mono/mono#12569]
Currently Mono.Cairo is included in the mono repository. Is it worth splitting it out into it's own repository and creating a nuget package from it.

I was recently working on a project on Windows that uses Mono.Cairo. I was getting compile errors because of a missing method. After investigating a bit I noticed that the version of Mono.Cairo that is included with the gtk-sharp msi installer for Windows included an older version of Mono.Cairo. This got me thinking that it'd be nice to have it split out and publish the assembly on nuget so that we could easily have a consistent version across all platforms and not have it tied to the mono repository.

Thoughts?

**Comment by marek-safar** [comment]
We usually do it only for code where we see a potential for future improvements and development. If the nuget packaging is the only "addition" to what we have today I'd much prefer to allow building the package from Mono than moving code around

**Comment by migueldeicaza** [comment]
Exactly, we can keep the code here, and just add the NuPkg files to do the build.

Additionally, we can setup easily an Azure Pipelines job that publishes new versions of the library.

But perhaps Mono.Cairo could get more action if it was split up, but it brings the burden on Mono to bring another submodule.
